roles/postgresql-configuration/templates/etc/postgresql/pg_version/main/pg_hba.conf.j2

```diff
@@ -87,7 +87,7 @@ local   all             postgres                                trust
 # TYPE  DATABASE        USER            ADDRESS                 METHOD
 
 # "local" is for Unix domain socket connections only
-local   all             all                                     peer
+local   all             all                                     md5
 # IPv4 local connections:
 host    all             all             127.0.0.1/32            trust
```

Comment:
Would be nice to cater for when Postgres is installed on a different host by making `127.0.0.1/32` a variable. I guess it works right now. Just a note